{"name": "OrangeHRM Login with Multiple parameters -- @1.3 ", "status": "failed", "statusDetails": {"message": "AssertionError: Test Failed\n", "trace": "  File \"C:\\Users\\HELLO\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\HELLO\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\orangehrmlogin_steps.py\", line 18, in homepage\n    assert False,\"Test Failed\"\n"}, "steps": [{"name": "When enter username \"adminxyz\" and password \"admin123\"", "status": "passed", "start": 1740454989955, "stop": 1740455000165}, {"name": "Then user must be successfully login into the system", "status": "passed", "start": 1740455000165, "stop": 1740455006068}, {"name": "And close browser", "status": "failed", "statusDetails": {"message": "AssertionError: Test Failed\n", "trace": "  File \"C:\\Users\\HELLO\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\HELLO\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\orangehrmlogin_steps.py\", line 18, in homepage\n    assert False,\"Test Failed\"\n"}, "start": 1740455006068, "stop": 1740455006176}], "parameters": [{"name": "username", "value": "adminxyz"}, {"name": "password", "value": "admin123"}], "start": 1740454981590, "stop": 1740455008352, "uuid": "38c9bf62-7f0f-4995-ae42-aecab294f876", "historyId": "a517e662cbfb951f74194a9c5b0ddd05", "fullName": "OramgeHRM Login: OrangeHRM Login with Multiple parameters", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "OramgeHRM Login"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}